spec/aws/active_job/sqs/configuration_spec.rb

```diff
@@ -41,13 +41,83 @@ module SQS
         it 'accepts YAML config with alias' do
           allow_any_instance_of(ERB).to receive(:result).and_return(<<~YAML)
             common: &common
-              default: 'https://queue-url'
+              default:
+                url: 'https://queue-url'
             queues:
               <<: *common
           YAML
           expect { Aws::ActiveJob::SQS::Configuration.new }.to_not raise_error
         end
 
+        context 'ENV set' do
+          Configuration::GLOBAL_ENV_CONFIGS.each do |config_name|
+            next if config_name == :config_file
+
+            describe "ENV #{config_name}" do
+              let(:env_name) { "AWS_ACTIVE_JOB_SQS_#{config_name.to_s.upcase}" }
+
+              let(:cfg) { Configuration.new }
+
+              before(:each) do
+                ENV[env_name] = 'env_value'
+
+                file_options = {}
+                file_options[config_name] = 'file_value'
+                allow_any_instance_of(Configuration)
```

Comment:
Can we make this behavioral instead of implementation specific? Perhaps write out a temp file and pass it into the config file option or set the environment for the config file. The session store hem does this.